paper/benchmarkset.tex

```diff
@@ -335,36 +339,49 @@ \subsubsection{Cucurbiturils}
 Thus, one group found a 6.4-6.8 kcal/mol dependence on salt concentration~\cite{hsiao_prediction_2014}, possibly impacting other studies as well~\cite{monroe_converging_2014}
 
 %Force field
-Despite these issues, CB7 appears to be at the point where careful studies can probe the true accuracy of our force fields~\cite{henriksen_computational_2015, gao_binding_2015, yin_toward_2015}, and the results can be sobering, with RMS errors in the binding free energies as high as 8 kcal/mol~\cite{henriksen_computational_2015, monroe_converging_2014}. More encouragingly, the values of $R^2$ values can be as high as 0.92~\cite{henriksen_computational_2015}.) 
+Despite these issues, CB7 appears to be at the point where careful studies can probe the true accuracy of our force fields~\cite{henriksen_computational_2015, gao_binding_2015, yin_toward_2015}, and the results can be sobering, with RMS errors in the binding free energies as high as 8 kcal/mol~\cite{henriksen_computational_2015, monroe_converging_2014}. More encouragingly, the values of $R^2$ values can be as high as 0.92~\cite{henriksen_computational_2015}. 
 Some force fields appear relatively worse than others~\cite{hsiao_prediction_2014, muddana_prediction_2012}.
 Calculated values are in many cases quite sensitive to details of force field parameters~\cite{monroe_converging_2014, moghaddam_new_2011, muddana_prediction_2012}. For example, modest modification of some Lennard-Jones parameters yielded dramatic improvements in calculated values~\cite{yin_toward_2015}, and host-guest binding data has, accordingly, been suggested as an input for force field development~\cite{yin_toward_2015, henriksen_computational_2015, gao_binding_2015}.
 Water structure around CB7 and calculated binding enthalpies also appear particularly sensitive to the choice of water model~\cite{rogers_role_2013, fenley_bridging_2014, gao_binding_2015}, and water is clearly important for modulating binding~\cite{nguyen_grid_2012}.
 The water model also impacts the number of sodium ions which must be displaced (in sodium-based buffer) on binding~\cite{gao_binding_2015, henriksen_computational_2015}.
 
 Despite its apparent simplicity, CB7 is still a challenging benchmark that can put important issues into high relief. For example, in SAMPL4, free energy methods yielded $R^2$ values from 0.1 to 0.8 and RMS errors of about 1.9 to 4.9 kcal/mol for the same set of CB7 cases. This spread of results across rather similar methods highlights the need for shared benchmarks. Potential explanations include convergence difficulties, subtle methodological differences, and details of how the methods were applied~\cite{muddana_sampl4_2014}. Until the origin of such discrepancies is clear, it is difficult to know how accurate our methods truly are. 
 
+% Changes
+% Used booktabs for formatting (may not be completely compatible with revtex...).
+% Used siunitx to format the numbers and set automatic alignment with uncertainty.
```

Comment:
This is beautiful; I'd never seen the SI unit module before. Definitely something to use again!